tests/test_l10n_download_links.py

```diff
@@ -7,74 +6,47 @@
 import pytest
 import requests
 
+pytestmark = [pytest.mark.nondestructive, pytest.mark.skip_selenium]
```

Comment:
Neither did I. For reference, here are the docs: https://pytest.org/latest/example/markers.html#marking-whole-classes-or-modules